apps/process_tpstream.cxx

```diff
@@ -165,29 +180,28 @@ int main(int argc, char const *argv[])
   auto tp_subsystem_requirement = daqdataformats::SourceID::Subsystem::kTrigger;
 
   auto channel_map = dunedaq::detchannelmaps::make_map(channel_map_name);
+
+  // Read configuration
+  std::ifstream config_stream(config_name);
+  nlohmann::json config = nlohmann::json::parse(config_stream);
+
+  // Only use the first plugin for now.
+  nlohmann::json ta_algo = config["trigger_activity_plugin"][0];
+  nlohmann::json ta_config = config["trigger_activity_config"][0];
+
+  nlohmann::json tc_algo = config["trigger_candidate_plugin"][0];
+  nlohmann::json tc_config = config["trigger_candidate_config"][0];
```

Comment:
Will this crash if someone only wants TA making and doesn't provide TCMaker config?
I would have this as if(tcmaking is on) tc_algo = config...